connectors/embeddings-vector-database/src/main/java/io/camunda/connector/model/embedding/models/BedrockEmbeddingModelProvider.java

```diff
@@ -44,7 +44,7 @@ public record Configuration(
               label = "Region",
               description = "AWS Bedrock region")
           String region,
-      @NotBlank
+      @NotNull
```

Comment:
🔧 Can we also remove `constraints = @TemplateProperty.PropertyConstraints(notEmpty = true)` here or does it still make sense? Also for the dimensions field below and for other dropdowns in the other classes.

Reply:
Most probably it doesn't make sense to keep the notEmpty constraints for dropdowns as there should always be one item selected.